### PR TITLE
Custom Labels for Operator Pod

### PR DESCRIPTION
**Pull request by David-Angel**
Give the ability to put custom labels on the pgo pod.

### CODE REVIEW

**Comment by David-Angel** [comment]
Details about this PR and why it was added are here:
https://github.com/CrunchyData/postgres-operator/issues/3524

File: helm/install/templates/manager-upgrade.yaml [hunk 1]
```diff
@@ -16,6 +16,9 @@ spec:
     metadata:
       labels:
         {{- include "install.upgradeLabels" . | nindent 8 }}
+        {{- if .Values.customLabels }}
```
Comment (genofire) [inline helm/install/templates/manager-upgrade.yaml:19]: indent of toYaml should be same then around (for easier read) and just write once `.Values.customLabels`

```suggestion
        {{- with .Values.customLabels }}
        {{- toYaml . | indent 8 }}
        {{- end }}
```

**Comment by genofire** [comment]
looks good to me - sadly i am a random person (not associated with CrunchyData).

**Comment by jmckulk** [comment]
@David-Angel I attempted to test this change, and it didn't work for me. 

```
$ helm template helm/install/
Error: YAML parse error on pgo/templates/manager-upgrade.yaml: error converting YAML to JSON: yaml: line 21: mapping values are not allowed in this context

Use --debug flag to render out invalid YAML
```

Since we will be merging devel and main for the coming release, I thought it would be easier to make some updates to this change and submit a pr to the devel branch. I'll close this pr in favor of #207

**Comment by David-Angel** [comment]
@jmckulk   Excuse for the late reply but it looks like you got it with the separate pull request.
https://github.com/CrunchyData/postgres-operator-examples/pull/207

Thank you for implementing this.

**Pull request closed without merging**